### PR TITLE
Update Grouping_DataObject_Sets.md

### PR DESCRIPTION
**Pull request by jeffreyguo**
Fixing grammar mistakes and broken link

### CODE REVIEW

File: docs/en/02_Developer_Guides/00_Model/How_Tos/Grouping_DataObject_Sets.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 The [api:SS_List] class is designed to return a flat list of records.
 These lists can get quite long, and hard to present on a single list.
-[Pagination](/templates/how_tos/pagination) is one way to solve this problem,
+[Pagination](/en/3.0/howto/pagination/) is one way to solve this problem,
```
Comment (dhensby) [inline docs/en/02_Developer_Guides/00_Model/How_Tos/Grouping_DataObject_Sets.md]: The links shouldn't include the language or the version number - it should just be `/howto/pagination/`

**Comment by dhensby** [comment]
Thanks, if you could just fix the two links, that'd be perfect!

File: docs/en/02_Developer_Guides/00_Model/How_Tos/Grouping_DataObject_Sets.md (hunk 1 above)
Comment (jeffreyguo) [inline docs/en/02_Developer_Guides/00_Model/How_Tos/Grouping_DataObject_Sets.md]: If it's "/howto/pagination/", then the whole url is "http://doc.silverstripe.org/howto/pagination/", which will give users message "We're sorry…
The page you are looking for does not exist, or might have moved."
The "Pagination" part is only working in version 3.0, that's why I added the special version.
Please refer to http://doc.silverstripe.org/en/developer_guides/model/how_tos/grouping_dataobject_sets/

Comment (dhensby) [inline docs/en/02_Developer_Guides/00_Model/How_Tos/Grouping_DataObject_Sets.md]: I'm pretty sure the https://github.com/silverstripe/silverstripe-docsviewer module adds the prefix part of the URL on. I haven't see this fully qualified type of URL in any other doc file before.

@wilr any ideas about why there are no pagination docs in the new set up?

**Comment by sminnee** [comment]
Hey @jeffreyguo could you fix up the links and rebase this, please?

**Comment by jeffreyguo** [comment]
@dhensby, the link issue is fixed in https://docs.silverstripe.org/en/3.1/developer_guides/model/how_tos/grouping_dataobject_sets/. I'm closing this PR.

**Pull request closed without merging**